src/ngraph/runtime/cpu/mkldnn_emitter.cpp

```diff
@@ -578,21 +578,26 @@ size_t MKLDNNEmitter::build_batchnorm_forward(const mkldnn::memory::desc& input_
                                               const mkldnn::memory::desc& mean_desc,
                                               const mkldnn::memory::desc& variance_desc,
                                               const double eps,
-                                              bool bn_training_flag)
+                                              bool bn_training_flag,
+                                              const mkldnn::post_ops& pops)
 {
     size_t input_index = build_memory_primitive(input_desc);
     size_t weights_index = build_memory_primitive(weights_desc);
     size_t result_index = build_memory_primitive(result_desc);
     size_t mean_index = build_memory_primitive(mean_desc);
     size_t variance_index = build_memory_primitive(variance_desc);
 
+    mkldnn::primitive_attr conv_attr;
+    conv_attr.set_post_ops(pops);
+
     if (bn_training_flag)
     {
         size_t batchnorm_index = insert_primitive(new mkldnn::batch_normalization_forward(
             {{mkldnn::prop_kind::forward_training,
```

Comment:
Can you point me to the constructor which takes post_ops, I dint find the relevant one in the bn constructor.

Reply:
also the workspace requirement is only needed for bprop ? just wanted to clarify.

Reply:
Yes, we are replacing the old GOEs with the new ones. This is the easiest. Otherwise, we have to deal with I/O views.

Reply:
@pruthviIntel it's on `primitive_desc`. This one is for Convolution, but batch normalization has the same one : 
https://github.com/intel/mkl-dnn/blob/master/examples/simple_net_int8.cpp#L114

Reply:
> also the workspace requirement is only needed for bprop ? just wanted to clarify. 


I think so. @jbobba  ?